docs/source/tutorials_source/platform/tutorial_active_learning.py

```diff
@@ -33,58 +33,80 @@
 
 6. Update the labeled set to include the newly chosen samples and remove them from the unlabeled set.
 
-In this tutorial, we use a k-nearest-neighbor classifier that predicts the class of a sample
-based on the class of the k nearest samples in the labeled set.
-We use the euclidean distance between a sample's embeddings as the distance metric.
-The advantage of such a classifier compared to CNNs is that it is very fast and easily implemented.
+In this tutorial, we use logistic regression on top of the embeddings as a classifier.
+This is the same as using the embedding model as a pretrained backbone
+and putting a single layer classification head on top of it while fine-tuning only the classification head
+on the labeled dataset, but keeping the backbone frozen.
+Since the embeddings are already computed, we can use them directly as input to the classification head.
+This saves computational time and makes the tutorial quick to walk through.
 
 What you will learn
 -------------------
 * You learn how an active learning loop is set up and which components are needed for it.
-* You learn how to perform active learning with Lightly.
+* You learn how to do active learning with Lightly.
 
-Requirements
+Define your dataset
 ------------
-- Make sure you are familiar with the command line tools described at :ref:`lightly-command-line-tool`.
 - To make the definition of the dataset for training the classifier easy,
-  we recommend using a dataset where the samples are grouped into folder according to their class.
+  we recommend using a dataset where the images are grouped in folders by class.
   We use the clothing-dataset-small. You can download it using
 
 .. code::
 
-    git clone https://github.com/alexeygrigorev/clothing-dataset-small.git
+    git clone https://github.com/alexeygrigorev/clothing-dataset-small.git ./clothing-dataset-small
```

Comment:
Do we need the `./clothing-dataset-small` argument? I thought `git clone ...` would create a folder with the slug which in this case would be `clothing-dataset-small`?

Reply:
No, we could remove it. I added it to show that our users can clone into a different folder, if they want to. Would you prefer to remove it?

Reply:
I would remove it. This has nothing to do with the actual tutorial. We just use git because we have to get the dataset. No need to make it look more complex than it is.